### DISCUSSION TITLE
Index Assigned for last 10 recent project files to jump quickly.

### DISCUSSION THREAD

**Discussion by saiemsaeed** [zed-industries/zed#24620] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Zed should have a specific index for the last 10 recent files to jump to them quickly. Like i normally do Ctrl-P to open project files and search for the project file even if its like on the 4th index instead of going to that file using the arrow keys, so instead of that there should be a specific index assigned to 10 recently file to jump to them. So ideally jump to the last 5th recent file should be something like. 

- Press Ctrl-P
- Press 5
- Boom last 5th recent file is opened.

<img width="561" alt="Screenshot 2024-07-01 at 6 20 08 PM" src="https://github.com/zed-industries/zed/assets/33630921/4e5c0a01-e76a-499c-928c-f5afe201c952">

**Comment by notpeter** [comment]
JetBrains has a dedicated modal for this (`cmd-e` / `ctrl-e` "Recent Files Pop-up")
<img width="500" alt="Screenshot 2024-07-02 at 12 45 54" src="https://github.com/zed-industries/zed/assets/145113/a3945913-85c4-422a-b652-60355d956d8d">

**Reply by saiemsaeed** [reply]
Having different modal, or having it in the same modal, the main aim is to have shortcuts to switch to recent files.
